tutorials/introductory/usage.py

```diff
@@ -446,6 +450,38 @@ def my_plotter(ax, data1, data2, param_dict):
 # numbers or dates.  If you pass 1000 strings, Matplotlib will think you
 # meant 1000 categories and will add 1000 ticks to your plot!
 #
+#
+# Additional Axis objects
+# ------------------------
+#
+# Plotting data of different magnitude in one chart may require
+# an additional y-axis. Such an Axis can be created by using
+# `~.Axes.twinx` to add a new Axes with an invisible x-axis and a y-axis
+# positioned at the right (analogously for `~.Axes.twiny`).
+#
+# Similarly, you can add a `~.Axes.secondary_xaxis` or
+# `~.Axes.secondary_yaxis` having a different scale than the main Axis to
+# represent the data in different scales or units.
+
+
+fig, (ax1, ax3) = plt.subplots(1, 2, figsize=(8, 2.7), layout='constrained')
+l1, = ax1.plot(t, s)
+ax2 = ax1.twinx()
+l2, = ax2.plot(t, range(len(t)), 'C1')
+ax2.legend([l1, l2], ['Sine (left)', 'Straight (right)'])
+
+ax3.plot(t, s)
+ax3.set_xlabel('Angle [°]')
+ax4 = ax3.secondary_xaxis('top', functions=(np.rad2deg, np.deg2rad))
+ax4.set_xlabel('Angle [rad]')
+
+##############################################################################
+# See :doc:`/gallery/subplots_axes_and_figures/two_scales`,
+# :doc:`/gallery/axisartist/demo_parasite_axes2` and
+# :doc:`/gallery/subplots_axes_and_figures/secondary_axis` for further examples
+# .
```

Comment:
Suggest working this in above in context, rather than a different paragraph.  

Also please do not link anything in `axisartist` in this tutorial.  Most people should not have to use the axis artist tool kit and we should not be steering people there if we can help it.

Reply:
OK, moved it up and removed demo_parasite_axes2.